templates/search.html

```diff
@@ -180,6 +180,7 @@ <h3 class="title">
                          height="20px">&nbsp;{{ state }}
                 {% endfor %}</span>
               </div>
+              <div>{{ res.created_dt }}</div>
```

Comment:
Always format date/time fields in templates, using a single project-specific formatter.
In this case it is date:"d F Y" (already commited)